process/modules/cycling_infra.py

```diff
@@ -221,7 +225,7 @@ def process(self):
         target_infra_polys = self._process_result_lines.copy()
         target_infra_polys = self._buffering(target_infra_polys)
 
-        target_infra_polys.rename(columns={"index_right1": "kadun_nimi"}, inplace=True)
+#        target_infra_polys.rename(columns={"index_right1": "kadun_nimi"}, inplace=True)
```

Comment:
This can be deleted, if no longer needed.

Reply:
Deleted